packages/framework/presence/src/test/notificationsManager.spec.ts

```diff
@@ -264,9 +264,12 @@ describe("Presence", () => {
 				false,
 			);
 
-			assert(eventHandlerCalls.original.length === 1);
-			assert(eventHandlerCalls.secondary.length === 1);
-			assert(eventHandlerCalls.tertiary.length === 1);
+			assert(eventHandlerCalls.original.length === 1, "original event handler was not called");
```

Comment:
I like a helpful message. It could be that event handler was called more than once.

Reply:
I included the actual value in the message so that should clarify.